exec-healthz/build/build.sh

```diff
@@ -16,6 +16,11 @@
 
 set -o errexit
 set -o nounset
+# For ubuntu updating bash to avoid illegal instruction error for pipefail
```

Comment:
yeah, this is a worrying code smell. if we really do need to install bash, it should be done somewhere else than in this script.